assets/text/effects/genericCampaignPlots_last_imperialPalace_capstone.properties

```diff
@@ -0,0 +1,15 @@
+#suppress inspection "UnusedProperty" for whole file
+.longName=Emperor
+.name=
+~01~prompt~panel_001~1_narration=It's been a fight to the palace door, but night has afforded <hero2.exists:them/<hero.mf:him/her/them>> some cover, and <hero2.exists:their/<hero>'s> skills have accomplished the rest.
+~01~prompt~panel_001~2_hero=<hero2.exists:<hero.bookish/coward/hothead/leader/poet/60:Drauven architecture rarely reaches this level of... [i]permanence[]./If those monsterskulls are meant \nto be intimidating... it's [i]working.[][face:interested]/[face:angry]How cute with those big skulls. Trying to be [i]scary?[]/Alright, it's intimidating. But we're not [i]un[]scary ourselves./<hero.drauven:[face:grim][small]The Drauven Emperor of old must have fallen, even if it was never told. Today, let us finish this tale. Write the final verse of his warsong./[face:grim][small][i]"Smile if you hunt the trophy-keeper,"[] writes <npc.fullname>. [i]"He's a sentimental fool, rife with appetites, easily fooled.">/You can't say the Emperor has no style.>/>
```

Comment:
A Drauven hero might have something different/insightful to say for `Drauven architecture rarely reaches this level of... [i]permanence[].`?